internal/controller/entra/main.go

```diff
@@ -97,3 +98,27 @@ func (c *Controller) IsStaffMember(ctx context.Context, username types.Username)
 
 	return isStaff, nil
 }
+
+func (c *Controller) SendInvite(ctx context.Context, email string, sponsor types.Sponsor) error {
+	requestBody := graphmodels.NewInvitation()
+	invitedUserEmailAddress := email
+	inviteRedirectUrl := config.EntraInviteRedirectURL()
+	sendInvitationMessage := true
+
+	requestBody.SetInvitedUserEmailAddress(&invitedUserEmailAddress)
+	requestBody.SetInviteRedirectUrl(&inviteRedirectUrl)
+	requestBody.SetSendInvitationMessage(&sendInvitationMessage)
+
+	message := "You have been invited to join the UCL ARC Portal by " + string(sponsor.ChosenName)
```

Comment:
potentially want a conditional on here if they've not set their chosen name then fallback to their username?